eng/pipelines/runtime.yml

```diff
@@ -334,6 +334,11 @@ jobs:
       extraStepsParameters:
         creator: dotnet-bot
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+        scenarios:
+        # Only one scenario per build for jobs which install xharness CLI
+        # until we fix https://github.com/dotnet/arcade/issues/5919
```

Comment:
https://github.com/dotnet/arcade/issues/5919 should be fixed now, but you might need to update the Helix SDK since Arcade wasn't released yet

Reply:
I tried bumping Arcade as part of this for testing and EVERYTHING EXPLODED so I'll wait for a bump to hit master I think

Reply:
@directhex bump Helix SDK, not arcade. Check the iOS draft PR and cherry pick that